### DISCUSSION TITLE
how to show hex startaddress in threads

### DISCUSSION THREAD

**Discussion by UnrealKaraulov** [winsiderss/systeminformer#2217] [Questions / Answers] [answered]
how to show hex startaddress in threads i don't see this option

![image](https://github.com/user-attachments/assets/a5a952a4-c014-4abb-afac-460818720888)

how to show hex instead of amdfendr.sys and etc?

**Comment by dmex** [comment, marked as answer]
You can uncheck `Enable undecorated symbols` from the Options > General tab to show addresses in hex

**Reply by UnrealKaraulov** [reply]
Not working, same as with this option, no hex in addresses.

No option to show 0xFFFF......00 instead of AgileVpn.sys and etc?

**Reply by dmex** [reply]
There's a missing check for the setting when opening the window... As a workaround you need to uncheck the View menu > `Refresh automatically` option before opening the process properties window and it'll show the addresses.

The next canary release will fix the setting so this workaround is only temporary.

**Reply by UnrealKaraulov** [reply]
@dmex not fixed
